packages/markdown-docx/src/OOXMLTransformer/index.js

```diff
@@ -58,200 +87,214 @@ class OoxmlTransformer {
     }
 
     /**
-     * Gets the value of the variable
+     * Constructs a ciceroMark Node for inline element from the information.
      *
-     * @param {Array} elements the variable elements
-     * @returns {string} the value of the variable
+     * @param {object} nodeInformation Contains properties and value of a node
+     * @return {object} CiceroMark Node
      */
-    getValue(elements) {
-        const variableContents = elements[1];
-        for (const property of variableContents.elements[0].elements) {
-            if (property.name === 'w:t') {
-                return property.elements[0].text;
+    constructCiceroMarkNode(nodeInformation) {
+        let obj = {};
+        if (nodeInformation.nodeType === 'softbreak') {
+            obj = {
+                $class: `${NS_PREFIX_CommonMarkModel}Softbreak`,
+            };
+        } else if (nodeInformation.nodeType === 'variable') {
+            if (nodeInformation.elementType && nodeInformation.name && nodeInformation.value) {
```

Comment:
I don't think so you need this check. There won't be any case where any of them is not there if a variable is present.

Reply:
There was a case present. This is the reason i used above.

Reply:
Is this case handled in any test case? If not, please push it.

Reply:
Actually, the test case file is deleted. I will try to recreate a similar test

Reply:
@algomaster99 
I have pushed a commit which involves the test for this.

Reply:
```json
{
  "$class": "org.accordproject.commonmark.Document",
  "xmlns": "http://commonmark.org/xml/1.0",
  "nodes": [
    {
      "$class": "org.accordproject.commonmark.Heading",
      "level": "2",
      "nodes": [
        {
          "$class": "org.accordproject.commonmark.Text",
          "text": "Try TemplateMark"
        }
      ]
    },
    {
      "$class": "org.accordproject.commonmark.Paragraph",
      "nodes": [
        {
          "$class": "org.accordproject.commonmark.Text",
          "text": "Just trying "
        },
        {
          "$class": "org.accordproject.commonmark.Strong",
          "nodes": [
            {
              "$class": "org.accordproject.commonmark.Text",
              "text": "few things "
            },
            {
              "$class": "org.accordproject.commonmark.Emph",
              "nodes": [
                {
                  "$class": "org.accordproject.commonmark.Text",
                  "text": "here"
                }
              ]
            }
          ]
        },
        {
          "$class": "org.accordproject.commonmark.Softbreak"
        },
        {
          "$class": "org.accordproject.commonmark.Strong",
          "nodes": [
            {
              "$class": "org.accordproject.commonmark.Text",
              "text": "hello"
            }
          ]
        },
        {
          "$class": "org.accordproject.commonmark.Text",
          "text": " "
        },
        {
          "$class": "org.accordproject.commonmark.Strong",
          "nodes": [
            {
              "$class": "org.accordproject.commonmark.Text",
              "text": "there"
            }
          ]
        }
      ]
    }
  ]
}
```

This is the test case you pushed but I cannot see any variable entity. So your program shouldn't even enter this `if` branch.

Reply:
To further prove my point, check t[his step](https://github.com/accordproject/markdown-transform/pull/421/checks?check_run_id=2982693335#step:6:1122) out. The `else` branch is not being covered also.